Makefile

```diff
@@ -180,11 +187,20 @@ ocaml-deps:
 	opam switch 4.03.1+k
 	eval `opam config env` && opam install -y mlgmp zarith uuidm cryptokit secp256k1.0.3.2 bn128 hex ocaml-protoc rlp yojson ocp-ocamlres bisect_ppx
 
+haskell-deps: kore-deps
+		git submodule update --init -- $(KORE_SUBMODULE)
```

Comment:
I don't want this to be a part of the Makefile. It should already be part of the rest of the install instructions.